pkg/webhook/risingwave_validating_webhook.go

```diff
@@ -58,6 +58,16 @@ func (v *RisingWaveValidatingWebhook) validateGroupTemplate(path *field.Path, gr
 		}
 	}
 
+	// Validate the cpu resources.
+	if groupTemplate.Resources.Limits.Cpu().Cmp(*groupTemplate.Resources.Requests.Cpu()) == -1 {
+		fieldErrs = append(fieldErrs, field.Required(path.Child("resources", "cpu"), "insufficient cpu resource"))
+	}
+
+	// Validate the memory resources.
+	if groupTemplate.Resources.Limits.Memory().Cmp(*groupTemplate.Resources.Requests.Memory()) == -1 {
+		fieldErrs = append(fieldErrs, field.Required(path.Child("resources", "memory"), "insufficient memory resource"))
+	}
+
```

Comment:
Sorry for not mentioning this before, but it seems that the following forms are all valid:

```yaml
resources:
   requests:
     cpu: 1
     memory: 100Mi 
```

```yaml
resources:
   limits:
     cpu: 1
   requests:
     memory: 100Mi 
```

That means we should do the comparison only when the limits exist.